willeswa/sendit-2
# #162155711 Fixes validation issues and cleans up database injections

#### What does this PR do?
Sanitizes data before injection into the database
#### Description of Task to be completed?
Capture errors that would make the application break
#### How should this be manually tested?
Run the app and enter inappropriate data such as empty strings. You should get an error invalidating the data
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/162155711
